storage/s3/src/main/java/io/aiven/kafka/tieredstorage/storage/s3/S3StorageConfig.java

```diff
@@ -48,6 +49,11 @@ public class S3StorageConfig extends AbstractConfig {
         + "To be used with custom S3-compatible backends (e.g. minio).";
     public static final String S3_REGION_CONFIG = "s3.region";
     private static final String S3_REGION_DOC = "AWS region where S3 bucket is placed";
+
+    public static final String S3_STORAGE_CLASS_CONFIG = "s3.storage.class";
+    private static final String S3_STORAGE_CLASS_DOC = "S3 storage class to store log segments";
```

Comment:
Actually not only segments but all objects related to it (indexes, manifest).
Maybe a bit more general description:
```suggestion
    private static final String S3_STORAGE_CLASS_DOC = "Defines which storage class to use when uploading objects";
```

Reply:
Don't forget to regenerate the docs 👍🏽